crates/iota-ledger-signer/examples/ledger_signer.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::str::FromStr;
+
+use clap::{Arg, Command};
+use iota_sdk::{
+    IotaClientBuilder,
+    types::{crypto::EncodeDecodeBase64, transaction::TransactionData},
+};
+
+fn transaction_from_base64(b64: &str) -> Result<TransactionData, anyhow::Error> {
```

Comment:
Feels like something that should be a method of `TransactionData` if not already

Reply:
the keytool command also does it like this:
```
let tx_bytes = Base64::decode(&tx_bytes).map_err(|e| anyhow!("Invalid base64 key: {:?}", e))?;
let tx_data: TransactionData = bcs::from_bytes(&tx_bytes)?;
```